packages/overlay/src/index.tsx

```diff
@@ -130,12 +132,13 @@ export async function init(options: SpotlightOverlayOptions = {}) {
   const finalExperiments = { ...DEFAULT_EXPERIMENTS, ...experiments };
 
   // Sentry is enabled by default
-  const defaultIntegrations = () => [sentry({ sidecarUrl })];
+  const defaultIntegrations = () => [sentry({ sidecarUrl: sidecarBaseUrl })];
```

Comment:
Since we have that context thing, do we still need to pass this as an init option?

Reply:
Yes, because this integration is exported, user can pass custom sidecar URL here as well.

Reply:
I understand but that doesn't sound something sensible to me.

Reply:
yeah, now we have sidecarurl in Spotlight context.

Removed sidecarurl from props -> https://github.com/getsentry/spotlight/pull/558/commits/9abb99156926d2e2e4ba920bddecd38297597f0c